test_scripts/API/PerformAudioPassThru/ideas_PAPT_new_param_audioPassThruIcon.txt

```diff
@@ -0,0 +1,56 @@
+  +./test_scripts/API/PerformAudioPassThru/001_ATF_P_PerformAudioPassThru_SUCCESS_All_Params.lua 
+  Check_that_SUCCESS_returned_when_all_parameters_valid_within_bounds
+  +./test_scripts/API/PerformAudioPassThru/002_ATF_P_PerformAudioPassThru_SUCCESS_All_Params_Upper.lua

```

Comment:
@maria-mih, I think this verification is redundant, since scope of req-t just to verify a new parameter for existing RPC. Verification of other parameters of this RPC is in scope of regression.